makeflow/src/dag_visitors.c

```diff
@@ -521,10 +521,10 @@ void dag_to_cyto(struct dag *d, int condense_display, int change_size)
 	fprintf(cytograph, "\t<att name = \"layoutAlgorithm\" value = \"Grid Layout\" type = \"string\" cy:hidden = \"1\"/>\n");
 
 	if(change_size) {
-		hash_table_firstkey(d->completed_files);
-		while(hash_table_nextkey(d->completed_files, &label, (void **) &name)) {
-			stat(label, &st);
-			average += ((double) st.st_size) / ((double) hash_table_size(d->completed_files));
+		hash_table_firstkey(d->files);
+		while(hash_table_nextkey(d->files, &name, (void **) &f) && dag_file_exists(f)) {
+			stat(name, &st);
```

Comment:
I agree that this stat may fail, though I don't believe this is different than what we were previously doing. The completed files structure was removed as we now rely on file state to determine if it was completed. Let me know if we should update this to an actual file check before the stat.